file-formats/siad/type/zif_aff_siad_v1.intf.abap

```diff
@@ -0,0 +1,43 @@
+INTERFACE zif_aff_siad_v1
+  PUBLIC.
+
+  TYPES:
+
+    "! <p class="shorttext">Assignment</p>
+    "! Assignment
+    BEGIN OF ty_assignment,
+
+      "! <p class="shorttext">Business Role Template ID</p>
+      "! Business Role Template ID
+      "! $required
+      BusinessRoleTemplateID   TYPE aps_iam_brt_id,
```

Comment:
we prefer primitive data types (towards self contained, assuming basic types as `char` are known), so `aps_iam_brt_id` is a `char[30]` right?

Reply:
same for `/UI2/SPACE_ENTITY_ID`

Reply:
I even guess you want to use underscores this will transform it to camel case in the JSON schema. Therefore, I suggest to use `business_role_template_id` or even `business_role_template`.

Since it is an object name with the length of 30 characters, you might want to reuse `zif_aff_types_v1=>ty_object_name_30`

```suggestion
      business_role_template_id   TYPE zif_aff_types_v1=>ty_object_name_30,
```